vignettes/cookbook_interpretation.Rmd

```diff
@@ -14,59 +14,164 @@ knitr::opts_chunk$set(
+
+If you use the `by_company = TRUE` option of the `target_market_share()` function, most of what's said above is true except relative to the company listed in that row in the `name_abcd` column.
+
+For detailed information about the table output of the `target_market_share()` function and its columns, see the [data dictionary](data_outputs.html#target_market_share-output).
+
+#### Interpretation & Intuition
+
+[2 or 3 sentences about interpretation]
+
+[2 or 3 sentences about intuition]
+
+### Technoloy Mix Plot
+
+The technology mix metric focuses on technology shifts within the power, fossil fuels and automotive sectors, namely: (i) the changes in the technological processes by which outputs are produced (e.g. shift from coal-fueled to renewable-fueled power capacity), and (ii) changes in the nature of the output itself (e.g. shift from internal combustion engines to electric vehicles). This metric measures the bank’s relative exposure to the economic activities that are impacted by the transition to a low-carbon economy.
+
+#### Explanation of Output
+
+[2 or 3 sentence explanation on plot output]
+
+#### Interpretation & Intuition
+
+[2 or 3 sentences about interpretation]
+
+[2 or 3 sentences about intuition]
+
+```{r, message=FALSE, warning=FALSE, echo=FALSE}
+library(pacta.loanbook)
+
+market_share_targets_portfolio <-
+  target_market_share(
+    data = prioritize(match_name(loanbook_demo, abcd_demo)),
+    abcd = abcd_demo,
+    scenario = scenario_demo_2020,
+    region_isos = region_isos_demo
+  )
+
+qplot_techmix(
+  dplyr::filter(
+    market_share_targets_portfolio,
+    scenario_source == "demo_2020",
+    sector == "power",
+    region == "global",
+    metric %in% c("projected", "corporate_economy", "target_sds")
+  )
+)
+
+```
+
+### Trajectory Plot
+
+The production volume trajectory metric aims to measure the alignment of a portfolio’s projected production volumes, based on the five-year capital plans of companies, to those given in climate scenarios. It is used for the fossil fuels, power, and automotive sectors. Changes in production volumes result either from transfer of production from one technology to another (e.g. internal combustion engines to electric vehicles) or from sheer expansion or contraction in the production coming from the technology/fuel (e.g. a company brings a new coal-fired power plant online). Projected production volumes at a 5-year horizon are considered at the individual client level at the technology level. The resulting volume trajectories are then compared with the trends set as targets in climate scenarios.
+
+#### Explanation of Output
+
+[2 or 3 sentence explanation on plot output]
```

Comment:
Again, explanation is missing.
Trajectory plot feels like it might actually need some type of explanation "falling between in a shaded regions means you are between the trajectory of scenarios X and Y, for the specific sector and technology... blah blah

Reply:
@jacobvjk added this